Makefile

```diff
@@ -8,40 +6,38 @@ else
 endif
 
 TARGETS=awesome
-BUILDLN=build
+BUILDDIR=build
 
-all: $(TARGETS) $(BUILDLN) ;
+all: $(TARGETS) ;
```

Comment:
Why? The dependency chain is `all` -> `awesome` -> `cmake-build` -> `build/CMakeCache.txt`. If `build` does not exist, then `build/CMakeCache.txt` does not exist and so the rule for `build/CMakeCache.txt` is run and `build` is created and `cmake` is run.